### PR TITLE
Add Intro shows

### PR DESCRIPTION
**Pull request by IsaackMwenda**
This were Keep II introduction shows meant to advertise the program. We may or may not analyze them but we want to know the engagement numbers.

### CODE REVIEW

File: code_schemes/dadaab_ws_correct_dataset.json [hunk 1]
```diff
@@ -135,6 +135,17 @@
         "dadaab s01e07"
       ]
     },
+    {
+      "CodeID": "code-3bf620g4",
+      "CodeType": "Normal",
+      "DisplayText": "dadaab s01_intro",
```
Comment (as2388) [inline code_schemes/dadaab_ws_correct_dataset.json]:
```suggestion
      "DisplayText": "dadaab s01 intro",
```

File: code_schemes/dadaab_ws_correct_dataset.json [hunk 2]
```diff
@@ -135,6 +135,17 @@
...
+      "StringValue": "dadaab_s01_intro",
+      "NumericValue": 23,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "dadaab s01_intro"
```
Comment (as2388) [inline code_schemes/dadaab_ws_correct_dataset.json]:
```suggestion
        "dadaab s01 intro"
```

File: src/lib/code_schemes.py [hunk 3]
```diff
@@ -25,6 +26,8 @@ class CodeSchemes(object):
     KAKUMA_S01E05_REASONS = _open_scheme("kakuma_s01e05_reasons.json")
     KAKUMA_S01E06_REASONS = _open_scheme("kakuma_s01e06_reasons.json")
     KAKUMA_S01E07_REASONS = _open_scheme("kakuma_s01e07_reasons.json")
+    KAKUMA_S01_INTRO_REASONS = _open_scheme("kakuma_s01_intro_reasons.json")
+
```
Comment (as2388) [inline src/lib/code_schemes.py:30]:
```suggestion
```

File: src/lib/pipeline_configuration.py [hunk 4]
```diff
@@ -181,6 +181,26 @@ class PipelineConfiguration(object):
...
+                           code_scheme=CodeSchemes.DADAAB_S01_INTRO_REASONS,
+                           coded_field="rqa_s01_intro_coded",
+                           analysis_file_key="rqa_s01_intro_",
+                           fold_strategy=lambda x, y: FoldStrategies.list_of_labels(CodeSchemes.DADAAB_S01_INTRO_REASONS,
+                                                                                    x, y)
+                       )
+                   ],
+                   ws_code=CodeSchemes.DADAAB_WS_CORRECT_DATASET.get_code_with_match_value("dadaab s01_intro"),
+                   raw_field_fold_strategy=FoldStrategies.concatenate),
+
```
Comment (as2388) [inline src/lib/pipeline_configuration.py]:
```suggestion
```

File: configurations/dadaab_pipeline_config.json [hunk 5]
```diff
@@ -12,7 +12,9 @@
         "wusc_keep_ii_s01_e04_dadaab_activation",
         "wusc_keep_ii_s01_e05_dadaab_activation",
         "wusc_keep_ii_s01_e06_dadaab_activation",
-        "wusc_keep_ii_s01_e07_dadaab_activation"
+        "wusc_keep_ii_s01_e07_dadaab_activation",
+        "wusc_keep_ii_s01_intro_dadaab_activation"
+
```
Comment (as2388) [inline configurations/dadaab_pipeline_config.json]:
```suggestion
```

File: code_schemes/dadaab_s01_intro_reasons.json [hunk 6]
```diff
@@ -0,0 +1,133 @@
+{
+  "SchemeID": "Scheme-b8ic3965",
```
Comment (as2388) [inline code_schemes/dadaab_s01_intro_reasons.json]: (Out of interest, how are you generating these? I think it's perfectly fine for these to be any valid string, but I thought we had a convention of restricting to hexadecimal. I'm questioning why and where that came from now though...)

File: code_schemes/kakuma_ws_correct_dataset.json [hunk 7]
```diff
@@ -135,6 +135,17 @@
         "kakuma s01e07"
       ]
     },
+    {
+      "CodeID": "code-daf99cu0",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma s01_intro",
```
Comment (as2388) [inline code_schemes/kakuma_ws_correct_dataset.json]:
```suggestion
      "DisplayText": "kakuma s01 intro",
```
I think, happy to discuss if you disagree

**Review by as2388 — APPROVED** [review]

File: code_schemes/dadaab_s01_intro_reasons.json (hunk 6 above)
Comment (IsaackMwenda) [reply code_schemes/dadaab_s01_intro_reasons.json]: Edited from previous a scheme. Ahh, there is a hexadecimal standard, now I know. Updated thanks.

**Pull request merged**